scripts/stubs.py

```diff
@@ -2,7 +2,8 @@
 from collections import namedtuple
 from typing import Dict, Tuple, Union
 
-Definitions = Union[str, Tuple[str, ...]]
+Subdefinitions = Union[str, Tuple[str, ...]]
```

Comment:
Do you mind chnging the case to `SubDefinitions`?

Reply:
No pb